src/Tree.scala

```diff
@@ -0,0 +1,16 @@
+/**
+  * Created by Zach on 10/17/16.
+  */
+class Tree {
+
+  var color = ""
+  var species = ""
+  var isEvergreen = true
+
+  def printFields(): Unit ={
+    var printEvergreen = ""
+    if (isEvergreen) printEvergreen = "is an Evergreen" else printEvergreen = "is not an Evergreen"
```

Comment:
in Scala the If-Else statement returns a value so just do this instead:
  printEvergreen = if (isEvergreen) "is an Evergreen" else "is not an Evergreen"